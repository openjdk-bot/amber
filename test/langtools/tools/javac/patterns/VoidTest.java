/*
 * @test /nodynamiccopyright/
 * @summary Ensure that void expressions passed to switch statement are rejected
 * @compile/fail/ref=VoidTest.out -XDrawDiagnostics VoidTest.java
 */
public class VoidTest {
    public static void test() {
        switch (test()) {
        }
    }
}
