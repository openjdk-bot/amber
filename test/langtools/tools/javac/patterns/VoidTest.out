VoidTest.java:8:16: compiler.err.prob.found.req: (compiler.misc.inconvertible.types: void, int)
1 error
